rpm/0008-Re-introduce-disable-passwdmgr.inc.patch

```diff
@@ -0,0 +1,36 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Daniel Suni <daniel.suni@jolla.com>
+Date: Fri, 17 Feb 2023 11:58:07 +0200
+Subject: [PATCH] Re-introduce disable-passwdmgr.inc
+
+Upstream has removed this file, but we still use it.
```

Comment:
Curious where and why. These look more like desktop linux password manager apps.

Reply:
Answering myself: the sailjail-permissions / Base.permissions in the initial commit.

@Tomin1 is that really intended for something?

Reply:
If they have removed it, I'd remove it from our's too. The content doesn't look that meaningful for us either. Those files should be invisible to apps anyway.

Reply:
Added PR for sailjail-permissions: https://github.com/sailfishos/sailjail-permissions/pull/138/files